docs/shared/guides/module-federation/module-federation-and-nx.md

```diff
@@ -0,0 +1,126 @@
+
+This can cause further issues if any of these third-party packages store internal state or use a Singleton to manage a single instance across your application. If a `remote` is deployed with a new version of the package, there is the possibility that your `federated module` will download a new copy of that package if your `host` does not have the same version it is expecting.
+
+This would then break the Singleton nature of the package as there will now be two instances of it running.
+
+{% callout type="note" title="Managing Versions" %}
+
+Follow our [Manage Library Versions Guide](/concepts/module-federation/manage-library-versions-with-module-federation) to learn more.
+
+{% /callout %}
+
+## Nx Support for Module Federation
+
+Nx offers out-of-the-box support for Module Federation with React and Angular. There are a number of features that can assist you when developing a Module Federation architecture for your application, such as:
+
+- Generators - to aid in scaffolding `remotes`, `hosts` and `federated modules`
+- Executors - to aid in building your applications with Module Federation and for great DX when developing locally
+- Type Safety - allowing for type-safety between `hosts` and `remotes` to catch issues early and to take advantage of autocompletion in IDEs
+- Versioning of Libraries - to aid in preventing some common issues regarding incompatible package versions being used by `federated modules`
+
+## Use Cases
+
+Nx has identified some common use cases that have made developers reach for Module Federation. They are Faster Builds and Independent Deployability.
+
+### Faster Builds
+
+As Module Federation allows you to split your application into smaller deployable chunks that are only required at runtime, you can take advantage of this to reduce the build times of your application.
+
+You can run the builds of multiple smaller applications in parallel and deploy all of them together, maintaining a single release cadence and co-ordination across teams but benefiting with reduced build times locally for developers and in CI.
+
+If you add [Nx Cloud](https://nx.app) to your Nx Workspace, then you can even get cache hits from some of the builds from other team members and CI, reducing the build time further.
+
+{% callout type="note" title="Faster Builds with Module Federation" %}
+
+Follow our [Faster Builds with Module Federation Guide](/concepts/module-federation/faster-builds-with-module-federation) to learn more.
+
+{% /callout %}
+
+### Independent Deployability
+
+Independent Deployability is the concept where individual teams within an organisation deploy their work on their own release cadence, regardless of other teams, allowing for more team autonomy. This can be achieved with Module Federation and becomes more and more appealing as the organisation and application grows.
```

Comment:
Minor: should we use US spelling `organization` or does this matter?

Reply:
lets stick to US